members/ILL000993.yaml

```diff
@@ -35,11 +35,11 @@ contact_form:
         - value: unsub2
           selector: "#feedback_mobile_opt_in"
         - value: Submit
-          selector: ".submit-button"
+          selector: input[type='submit'][value='Send feedback']
```

Comment:
do any more of these IL yamls need this updated?

Reply:
I've seen more IL targets updating their forms from something like this, http://www.ilhousegop.org/contacthays, to this http://www.ilhousegop.org/windhorst_contact which is when `submit` has been updated. Not all of them have made the full transition tho, thus why I've been updating these on a case by case basis. 

Short answer: yes but not right now